### PR TITLE
Updated doPost so that if the user inputs an image URL, the submitted message...

### PR DESCRIPTION
**Pull request by dayhahm**
will display the image, not the image URL. user-page.css has also been updated so that the images scale to an appropriate size for the message box.

Need to add extra logic in to check the validity of the image url (does the image url actually lead to an image).

### CODE REVIEW

**Review by Open-Fuse — APPROVED** [review]
Looks good!

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -78,7 +80,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
-    Message message = new Message(user, text);
+    //replace user inputted text if it fits the defined patten into an img tag
+    String regex = "(https?://\\S+\\.(png|jpg))";
```
Comment (NickChapman) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:84]: I would consider expanding this to include `gif`, `bmp`, and `jpeg` (not the extra e).

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 2]
```diff
@@ -78,7 +80,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+    String replacement = "<img src=\"$1\" />";
+    String replacedText = text.replaceAll(regex, replacement);
+
+    //***need to add url validation below***
```
Comment (NickChapman) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:88]: A common format for these kind of comments is `TODO(dayhahm): Add URL validation here.` That way it's easy to track down who this comment is left for. Another common format is `TODO(issue #): Add URL validation here` where `issue #` corresponds to an entry in your project tracking software.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 1 above)
Comment (dayhahm) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java:84]: will do in the next iteration!

**Pull request merged**